src/Tests/Microsoft.NET.TestFramework/Commands/NuGetRestoreCommand.cs

```diff
@@ -4,7 +4,6 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using Microsoft.DotNet.Cli.Utils;
```

Comment:
I all for removing unused usings, but please revert and do it as a separate change. It is not at all related to moving to repo toolset.